locale/en/LC_MESSAGES/django.po

```diff
@@ -4321,29 +4347,37 @@ msgstr ""
 "(e.g. a digital discussion group), and takes part actively in the discussion "
 "and/or collects data which can be traced back to individuals."
 
-#: reviews/utils/review_utils.py:435
+#: reviews/utils/review_utils.py:447
 msgid "De aanvraag bevat het gebruik van {}"
 msgstr "The application makes use of {}"
 
-#: reviews/utils/review_utils.py:456
+#: reviews/utils/review_utils.py:468
 msgid ""
 "De aanvraag bevat psychofysiologische metingen bij kinderen onder de {} jaar."
 msgstr ""
 "The application contains psychophysiological measurements on children under "
 "{} year(s) old."
 
-#: reviews/views.py:60
+#: reviews/views.py:61
 msgid "Mijn besluiten"
 msgstr "My results"
 
-#: reviews/views.py:82
+#: reviews/views.py:83
 msgid "Openstaande besluiten commissieleden"
 msgstr "Pending decisions committee members"
 
-#: reviews/views.py:200
+#: reviews/views.py:201
 msgid "Openstaande besluiten eindverantwoordelijken"
 msgstr "Pending decisions supervisors"
 
+#: reviews/views.py:567
+msgid ""
+"Deze aanvraag is al beoordeeld, dus je kan je beoordeling niet meer "
+"toevoegen/aanpassen"
+msgstr ""
+"This application has already been assessed, so you cannot add/adjust your "
+"assessment"
```

Comment:
misschien `add/edit`

Reply:
Looking back at it, I'd say `add/change'?